exe/sufia_import

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+require 'rails'
+require ::File.join(File.dirname(Bundler.default_gemfile), 'config/environment')
+
+options = OpenStruct.new
+options.verbose = false
+options.import_dir = "tmp/import"
+# TODO: add collection like options.mappings = ["Collection=collection_", "GenericFile=generic_file_"]
+options.mappings = ["GenericFile=generic_file_"]
+
+args = OptionParser.new do |opts|
+  opts.banner = "imports model metdata json files in, contacting fedora to import binaries. Usage: sufia_import [options]"
+
+  opts.on("-v", "--[no-]verbose", "Run verbosely") do |v|
+    options.verbose = v
+  end
+
+  opts.on("-d", "--json_directory [path/to/files]", "Directory containing files for import.  Defaults to tmp/import") do |dir|
```

Comment:
I assumed it would be moved, I guess because I am setting up a new box. I can change it if you have strong feelings, but considering people will probably have all kinds of different approaches, I like that at least the semantics of the location match those of the script.